scrowl-io/scrowl
# Sc 137

### Short summary

- Restructured the folder setup to better align with turborepo's setup for mono repo
   - Electron app is now under `apps` folder 
- Updated the the docker image as well as the compose files to better handle monorepo.
   - The image size actually decreased a bit!
      -  From 461.37 MB to 450.07 MB on the registry.
- The Makefile has bee updated to handle the changes to the folder structure.
- Added a new wiki page as well a minor update to the setup wiki page to that links to the new page
  - https://github.com/EEBOS/SCROWL/wiki/2.-VS-Code-settings 

---

### Test steps

1. Essentially nothing has changed in terms the project commands. 
   1. Use `make start` or `make cleanStart` and it should work as if nothing has changed. 

---

###  Note
1. All of the tsConfig files are in a centralized place under `packages/tsconfig`
1. All of the linters/formaters configs are in a centralized place under `packages/config`
1. I have kept the `packages/ui` folder for the time being in case the need came up. We can always remove it later on.
1. Please add the following to your `.vscode/settings.json` : https://github.com/EEBOS/SCROWL/wiki/2.-VS-Code-settings
